docs/reference/settings/security-settings.asciidoc

```diff
@@ -1281,8 +1281,8 @@ at the OpenID Connect Provider
 
 `rp.redirect_uri`::
 The Redirect URI within {kib}. Typically this is the
```

Comment:
```suggestion
The Redirect URI within {kib}. Typically, if you want to use the authorization code flow, this is the
```

Reply:
Maybe we can lose "typically" here ? It made more sense when we only had one URL to point to in the docs. Now we have two ( granted, with the same prefix ) so there is no "typical" scenario

Reply:
Yeah, makes sense, update suggestion to remove `Typically`